docs/iris/src/whatsnew/1.13.rst

```diff
@@ -1,34 +1,34 @@
-What's New in Iris 1.12
+What's New in Iris 1.13
 ***********************
 
-:Release: 1.12
+:Release: 1.13
 :Date: 2017-05-17
 
 
-This document explains the new/changed features of Iris in version 1.12
+This document explains the new/changed features of Iris in version 1.13
 (:doc:`View all changes <index>`.)
 
 Iris 1.13 Features
 ==================
 
 * Allow the reading of NAME trajectories stored by time instead of by particle number.
-* An experimental link to python-stratify via :mod:`iris.experimental.stratify`
+* An experimental link to python-stratify via :mod:`iris.experimental.stratify`.
 * Data arrays may be shared between cubes, and subsets of cubes, by using the :meth:`iris.cube.share_data` flag.
 
 
 Bug Fixes
 =========
 
 * The bounds are now set correctly on the longitude coordinate if a zonal mean diagnostic has been loaded from a PP file as per the CF Standard.
-* netCDF loading will now determine whether there is a string-valued scalar label, i.e.: a character variable that only has one dimension (the length of the string) and interpret this effectively.
+* NetCDF loading will now determine whether there is a string-valued scalar label, i.e. a character variable that only has one dimension (the length of the string), and interpret this correctly.
```

Comment:
'effectively' doesn't feel like the right word to me, but that may just be my interpretation of the word.